categories.json

```diff
@@ -505,12 +505,19 @@
           "areas": [ "DE", "AT", "CH" ],
           "form_details": [ "delivery", "access" ]
         },
+        "furniture": {
+          "name": "Furniture",
+          "osm_tags": [
+            { "shop": "furniture" }
+          ],
+          "areas": [ "DE", "AT", "CH" ]
+        },
         "gift": {
```

Comment:
You should remove it I believe as the SQL generated will have no effects.

Reply:
My rationale was to keep it to make it explicit that we don't show them. Otherwise we should probably remove the locale text and the icon as well. I don't think it has any negative effect on the generated SQL, or does it?

Reply:
I'm not very confident on the SQL side yes. I prefer to remove everything and revert the change if we change our mind later.